collections/cnes-land-cover-map/README.md

```diff
@@ -0,0 +1,177 @@
+# CNES Land Cover Map
+
+## Short description
+The CNES Land Cover Map (Occupation des Sols, OSO) produces land classification for Metropolitan France at 10 m spatial resolution based on Sentinel-2 L2A data within the Theia Land Cover CES framework. Maps for 2020, 2019, and 2018 use a 23-categories nomenclature. For earlier maps in 2017 and 2016, a fully compatible 17-classes nomenclature is employed.
+
+## Band information
+CNES Land Cover product has 3 bands. The `OCS` band contains main classification data with class values ranging from 1 to 23. The Land Cover nomenclature and colouring scheme is shown below.
```

Comment:
- Consider adding an article at the beginning of the sentence and use the verb consist or contain: "CNES Land Cover product has 3 bands." ➡️ The CNES Land Cover product contains 3 bands.